edge_intelligence_models/README.md

```diff
@@ -1,3 +1,5 @@
 # Edge Intelligence Models
 
-Place holder for Edge intelligence models
\ No newline at end of file
+Saved pfa, sklearn and tensorflow models are present under the windmill-models directory.
```

Comment:
Trained pfa ...

<PFA, SKlearn , Tensorflow link>
The generated data is using the TI sensor tag which has been used for training the windmill model.
It provides a basic example which uses TI sensor tag <link> data to simulate a sample windmill model. The basic metrics we collect from Windmill model are: